src/app/components/Lightsaber.js

```diff
@@ -0,0 +1,25 @@
+function Lightsaber() {
+  const root = document.querySelector('.remaining-time');
+  const divElement = document.createElement('div');
+  const handle = document.createElement('div');
+  const blade = document.createElement('div');
+  const redBlade = document.createElement('div');
+
+  divElement.classList.add('remaining-time__lightsaber');
+  divElement.classList.add('lightsaber');
```

Comment:
While adding classes to one element, separate classes with ','

`divElement.classList.add('remaining-time__lightsaber', 'lightsaber');`

Reply:
Also consider refactor with elementFactory